CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for crash
 Unreleased
 ==========
 
+ - Removed printing of full stack trace in case of connection error.
```

Comment:
Print only error message instead of full stacktrack in case of connection error.